packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -988,11 +994,37 @@ class SelectableRegionState extends State<SelectableRegion> with TextSelectionDe
       selectionGeometry: _selectionDelegate.value,
       onCopy: () {
         _copy();
-        hideToolbar();
+
+        // In Android copy should clear the selection.
+        switch (defaultTargetPlatform) {
+          case TargetPlatform.android:
+            _clearSelection();
+            break;
+          case TargetPlatform.iOS:
+            hideToolbar(false);
+            break;
+          case TargetPlatform.fuchsia:
```

Comment:
Here I think we should move fuchsia up with android.  I think it's best for Fuchsia to match Android until we have some kind of spec for any different behavior for it.

Reply:
done~